src/sentry/notifications/helpers.py

```diff
@@ -90,10 +114,10 @@ def should_be_participating(
 
 
 def where_should_be_participating(
-    user: Any,
-    subscriptions_by_user_id: Mapping[int, Any],
+    user: "User",
```

Comment:
@mgaeta we want a literal `"User"` instead of `User`? I don't know Python typings that much, but that seems odd to me

Reply:
I can't import `User` at the top of this file without causing a circular dependency. The [workaround](https://mypy.readthedocs.io/en/latest/runtime_troubles.html#import-cycles) it to use the `TYPE_CHECKING` block with quoted types.

Reply:
Ah, didn't know about this feature. Good to know!